src/packet.js

```diff
@@ -77,7 +78,8 @@ class Packet {
 				errorCode: this.rmcMessage.errorCode,
 				body: this.rmcData.body
 			},
-			stackTrace: this.stackTrace
+			stackTrace: this.stackTrace,
+			date: this.date
```

Comment:
`this.date` should also be initialized in the constructor

Reply:
If I were to replace line 52 in the constructor with `this.date = date;`, it causes the list to no longer populate when opening a file. If I were to use `date = this.date;`, the variable goes unused. Should I replace it with `this.date = 0;`?

Reply:
Yes `this.date = 0;` should be fine